frontend/public/components/app.jsx

```diff
@@ -128,7 +128,7 @@ class App extends React.PureComponent {
       <div id="content">
         <Route path={namespacedRoutes} component={NamespaceSelector} />
         <GlobalNotifications />
-        <div className="content-scrollable">
+        <div id="content-scrollable">
```

Comment:
Is it possible to only add the scrollable div in the overview instead of every page?

Reply:
It is possible but that will introduce inconsistent scrolling behavior regarding the project selector. The `content-scrollable` doesn't contain the project selector, meaning on the overview it will always be visible and the rest of the remaining pages it will be within the scrollable area.

Reply:
I found one and currently the modal body doesn't have scroll enabled, so that's a bug.

![modal-from-overview](https://user-images.githubusercontent.com/1874151/46227527-7c3b3600-c32d-11e8-88a4-619fa10aa2ee.gif)

When scroll is enabled I'm not see the "background" panels scroll effected by the modal.
![chrome-scroll-chaining-overview-sidenav](https://user-images.githubusercontent.com/1874151/46227587-b3a9e280-c32d-11e8-8ea9-a3d35061b6c6.gif)

Reply:
I don't know if this was a problem before, but I can scroll content behind dialogs on my phone. For instance, open the "Create Project" dialog. You can scroll the page behind the dialog content on iOS.

Reply:
OK I was able to reproduce on 3.11 as well, so that's not new. We should open a bug to track.

Reply:
Looks like I already opened a but for this, CONSOLE-658.